src/PresentationalComponents/Common/Tables.js

```diff
@@ -1,6 +1,8 @@
 import { generateFilter } from '@redhat-cloud-services/frontend-components-utilities/helpers';
 import { SYSTEM_TYPES } from '../../AppConstants';
 
+// All this feels like it would be better in a helpers file or something.
```

Comment:
This file is basically just helper functions used across the app. I might just rename to 'TableHelpers'